python-package/rgf/rgf_model.py

```diff
@@ -263,6 +276,11 @@ def _validate_params(self,
             raise ValueError(
                 "memory_policy must be 'conservative' or 'generous' but was %r." % memory_policy)
 
+        if not isinstance(init_model, (type(None), six.string_types)):
```

Comment:
How about
`if init_model is not None and not isinstance(init_model, six.string_types):` ?

Reply:
OK, but let's do this in a separate PR to not pollute this one, because it's not the only place where `type(None)` is used.
https://github.com/RGF-team/rgf/blob/0dbf0b6398d25312a338e347232d5b6b14ed82fc/python-package/rgf/rgf_model.py#L199
https://github.com/RGF-team/rgf/blob/0dbf0b6398d25312a338e347232d5b6b14ed82fc/python-package/rgf/rgf_model.py#L222

Reply:
OK!